client/lib/user-agent-utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function currentUserAgent() {
+	return navigator.userAgent;
+}
+
+export function isChromeOS( userAgent = currentUserAgent() ) {
+	return /\bCrOS\b/.test( userAgent );
+}
```

Comment:
I'm still a bit hesitant about this file. It only really provides one function, practically speaking. My worry is that it's not very discoverable and will thus likely sit for a very long time as a single-function-module dependency used by one component, at which point it wouldn't bring any benefit to simply inlining the function.

You might check if there are other places in the code where this could be useful. If there are many, maybe we can come around in a separate PR and reuse the code, but barring that effort, it might be just as good to create this function inside of the file with the component that imports it